word_manager/services/gpt_service.py

```diff
@@ -25,17 +25,17 @@ def send_prompt(manual_mode=False, simulate_error=None):
             return {"choices": [{"message": {"content": "Sample response for testing."}}]}
 
         content_message = f"Request:\n"
-        content_message += f"Word in Japanese: Give me a useful word to know"
-        word_objects = Word.objects.filter(language="Japanese")
+        content_message += f"Word in {settings.TARGET_LANGUAGE}: Give me a useful word to know"
+        word_objects = Word.objects.filter(language=settings.TARGET_LANGUAGE)
         word_list = [word.word for word in word_objects]
         if word_list:
             word_string = ', '.join(word_list)
             content_message += f" that is not in this list: [{word_string}]"
-        content_message += f".\nDefinition in Japanese: Give the definition of the word in Japanese with furigana.\n"
-        content_message += f"Context Sentence in Japanese: Offer a sentence using the word in Japanese with furigana.\n"
-        content_message += f"\nResponse Structure:\nWord: [In Japanese]\n"
-        content_message += f"Definition: [In Japanese with furigana]\n"
-        content_message += f"Context Sentence: [In Japanese with furigana]\n"
+        content_message += f".\nDefinition in {settings.TARGET_LANGUAGE}: Give the definition of the word in {settings.TARGET_LANGUAGE} with furigana (if applicable).\n"
+        content_message += f"Context Sentence in {settings.TARGET_LANGUAGE}: Offer a sentence using the word in {settings.TARGET_LANGUAGE} with furigana (if applicable).\n"
+        content_message += f"\nResponse Structure:\nWord: [In {settings.TARGET_LANGUAGE}]\n"
+        content_message += f"Definition: [In {settings.TARGET_LANGUAGE} with furigana (if applicable)]\n"
+        content_message += f"Context Sentence: [In {settings.TARGET_LANGUAGE} with furigana (if applicable)]\n"
```

Comment:
Try and rework this a tad. I'd like something like a conditional for the furigana part, maybe even a switch case based on the languages if you want to support the possibility of more language nuances.

So to summarize, just add a conditional that omits the 'with furigana' part if it's not Japanese and then apply the period as usual.

Reply:
This is probably outside of AC, but you went the extra mile to fully implement Spanish so lets take it all the way home haha

Reply:
Awesome, I just pushed commit `2b248a` that I think addresses this issue.